operator/api/v1alpha1/keptnworkload_types.go

```diff
@@ -29,6 +30,7 @@ import (
 type KeptnWorkloadSpec struct {
 	AppName                string            `json:"app"`
 	Version                string            `json:"version"`
+	TraceContext           map[string]string `json:"tracecontext,omitempty"`
```

Comment:
Is a map ok in this case, or do we have more structure information about this?

Reply:
I was debating a lot on this. In the end, I decided to go with a simple map because OTel does a good job of hiding the information about how to propagate the TraceContext. I decided to use a simple Map because OTel works with that and we delegate to it all the TraceContext handling.